setup.py

```diff
@@ -1,36 +1,39 @@
 import os
 import sys
-from setuptools import setup, find_packages
 
+from setuptools import setup, find_packages
 from setuptools.command.build_py import build_py
 from setuptools.command.install import install
+
 from build_manpages import __version__
-from build_manpages.build_manpages \
-    import build_manpages, get_build_py_cmd, get_install_cmd
-
-ld = """
-Generate manual page an automatic way from ArgumentParser object, so the manpage
-1:1 corresponds to the automatically generated --help output.  The manpage
-generator needs to known the location of the object, user can specify that by
-(a) the module name or corresponding python filename and (b) the object name or
-the function name which returns the object.  There's a limited support for
-(deprecated) optparse objects, too.
-"""
+from build_manpages.build_manpages import build_manpages
+from build_manpages.build_manpages import get_build_py_cmd
+from build_manpages.build_manpages import get_install_cmd
+
+
+ROOT_DIR = os.path.dirname(os.path.realpath(sys.argv[0]))
+
+
+def get_readme():
+    with open(os.path.join(ROOT_DIR, 'README.rst')) as fh:
```

Comment:
This file doesn't exist.

Reply:
This particular commit was rushed. Have dropped it from this PR to focus on the thing I actually want.